src/routes/maps-of-the-week/[page]/+page.server.ts

```diff
@@ -74,15 +74,18 @@ export async function load({ fetch, params }: LoadFunctionParameter): Promise<Ma
             map: {
                 id: singleMapOfTheWeek.mapId,
                 name: beatSaverMapData.name,
-                coverUrl: coverUrl,
+                coverUrl: coverUrl!!,
```

Comment:
What does post-variable `!!` do?
I don't think I know this language feature. I only know the non-null/undefined assertion with one `!`.

Reply:
That might have been my Kotlin brain...
Anyway, I already fixed that in one of my unpushed commits.